py3status/modules/spotify.py

```diff
@@ -91,9 +95,10 @@ class Py3status:
         "stereo",
         "version",
     ]
+    dbus_client = "org.mpris.MediaPlayer2.spotify"
```

Comment:
please add this in the alphabetical order within the available config params above

Reply:
done.